node/subsystem-types/src/messages.rs

```diff
@@ -130,6 +133,8 @@ pub enum CandidateValidationMessage {
 		ValidationCode,
 		CandidateDescriptor,
 		Arc<PoV>,
+		Duration,
+		/// Execution timeout
```

Comment:
rustfmt considers comments on the same related to the next line...

Reply:
I see, so fmt moved this around.